include/vk/vkwrap/surface.h

```diff
@@ -3,6 +3,8 @@
 #include "common/glfw_include.h"
 #include "common/vulkan_include.h"
 
+#include "window/window.h"
+
```

Comment:
Untangle the headers. This file does not have to include any window related files